tests/fixtures/file-ignores/combined/.gitignore

```diff
@@ -0,0 +1,4 @@
+/gitignore-root-excluded
```

Comment:
q: Did you forget to commit the files here because the `.gitignore` excluded them? 😅

Reply:
Nope, we're testing the rule collection in isolation so don't actually care about any project files in there, only the exclude and ignore files.

Reply:
Ahh gotcha, makes sense